app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  def show
+    @user = User.find_by(id: params[:id])
+    @mentor = User.find_by(id: @user.matched_id)
+
+  end
+  def dashboard
+    @mentor = User.find_by(id: current_user.matched_id)
+  end
+  def missing_mentor
+    @user = current_user
+    if !@user.blank? 
```

Comment:
Hi, for show page i think mentor can also see the page. So I dont need to check in for show action that it should be organiser and mentee. right ?